rules/number-literal-case.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const fix = value => {
+	if (!/^0[a-zA-Z]/.test(value)) {
+		return value;
+	}
+
+	const indicator = value[1].toLowerCase();
+	const val = value.substring(2).toUpperCase();
```

Comment:
I rarely use either of those. I personally have to experiment with or find the docs for `slice` every time I use it, while substring is more self-explanatory.
But I have a weak opinion on this. Feel free to write a proposal